packages/react-aria-components/docs/NumberField.mdx

```diff
@@ -128,7 +128,7 @@ the stepper buttons. `NumberField` helps achieve accessible number fields that s
 formatting options and can be styled as needed.
 
 * **Customizable formatting** – Support for internationalized number formatting and parsing including decimals, percentages, currency values, and units. Multiple currency formats are supported, including symbol, code, and name in standard or accounting notation.
-* **International** – Support for the Latin, Arabic, and Han positional decimal numbering systems in [over 30 locales](internationalization.html#supported-locales). The numbering system is automatically detected from user input, and input method editors such as Pinyin are supported.
+* **International** – Support for the Latin, Arabic, and Han, Devanagari and Bengali positional decimal numbering systems in [over 30 locales](internationalization.html#supported-locales). The numbering system is automatically detected from user input, and input method editors such as Pinyin are supported.
```

Comment:
```suggestion
* **International** – Support for the Latin, Arabic, Devanagari, Bengali, and Han positional decimal numbering systems in [over 30 locales](internationalization.html#supported-locales). The numbering system is automatically detected from user input, and input method editors such as Pinyin are supported.
```

from what I can tell, Devanagari and Bengali are already positional decimal numbering systems, where as Han is not necessarily, which is why we clarify that we support that particular style of it